cdap-cli/src/main/java/io/cdap/cdap/cli/command/ConnectCommand.java

```diff
@@ -69,6 +70,6 @@ public String getPattern() {
 
   @Override
   public String getDescription() {
-    return "Connects to a CDAP instance";
+    return "Connects to a CDAP instance (Deprecated, use connect-link instead)";
```

Comment:
mention deprecation in Class comment too for readability. Also mention to use ConnectLinkCommand instead in the class comment.